diff --git a/CMakeLists.txt b/CMakeLists.txt
index b9df1312..e6cc9507 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -479,9 +479,9 @@ if(OCTOTIGER_WITH_CUDA)
   endif()
 else()
   if (OCTOTIGER_WITH_FAST_FP_CONTRACT)
-        set_source_files_properties(src/unitiger/hydro_impl/hydro_kernel_interface.cpp PROPERTIES COMPILE_FLAGS "-march=native -ffp-contract=fast")
+        set_source_files_properties(src/unitiger/hydro_impl/hydro_kernel_interface.cpp PROPERTIES COMPILE_FLAGS "-ffp-contract=fast")
   else()
-        set_source_files_properties(src/unitiger/hydro_impl/hydro_kernel_interface.cpp PROPERTIES COMPILE_FLAGS "-march=native -ffp-contract=off")
+        set_source_files_properties(src/unitiger/hydro_impl/hydro_kernel_interface.cpp PROPERTIES COMPILE_FLAGS "-ffp-contract=off")
   endif()
 endif()
 
